setup.py

```diff
@@ -51,7 +51,9 @@
         'Products.CMFCore',
         'setuptools',
         'ftw.upgrade',
-        'path.py<9.0'
+        'path.py<9.0',
+        'plone4.csrffixes',
```

Comment:
Then I'll have to slap the CSRF check in place of that there and return 403 before the zip payload is generated. This requires me to fix most of the tests.